pkgs/applications/editors/leo-editor/default.nix

```diff
@@ -53,13 +54,15 @@ stdenv.mkDerivation rec {
     makeWrapper ${python3.interpreter} $out/bin/leo \
       --set PYTHONPATH "$PYTHONPATH:$out/share/leo-editor" \
       --add-flags "-O $out/share/leo-editor/launchLeo.py"
+
+    wrapQtApp $out/bin/leo
```

Comment:
Is it wrapped twice eventually? I'm not too familiar with these wrapping methods but if the answer is yes, then you may want to try out using either of them or perhaps the `makeWrapperArgs` array.